plugins/obs-webrtc/data/locale/fr-FR.ini

```diff
@@ -1,5 +1,7 @@
 Output.Name="Sortie WHIP"
 Service.Name="Service WHIP"
 Service.BearerToken="Jeton du porteur"
+Service.STUNServer="Serveur STUN"
+Service.WaitGatheredCandidates="Attendre les candidats ICE locaux"
```

Comment:
translations are not done in the pull requests, they are done in crowdin

Reply:
noted, thanks